core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStoreMessageHandler.java

```diff
@@ -42,6 +42,10 @@ public KafkaStoreMessageHandler(KafkaSchemaRegistry schemaRegistry,
    */
   @Override
   public void handleUpdate(SchemaRegistryKey key, SchemaRegistryValue value) {
+    if (value == null) {
```

Comment:
Should we also check for a key? How key looks like in case of a tombstone?

Reply:
Keys are always non-null.   Tombstones are always of the form (key, null).

Reply:
Maybe just add a debug statement?